src/main/java/org/alliancegenome/curation_api/model/entities/Identifier.java

```diff
@@ -0,0 +1,13 @@
+package org.alliancegenome.curation_api.model.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class Identifier {
+	private Long counter;
+	private String subdomain_code;
+	private String subdomain_name;
+	private String curie;
```

Comment:
Using the library will prevent us from having duplicate code between the two repos.